_data/backup.yml

```diff
@@ -57,9 +57,10 @@ websites:
     img: evernote.png
     tfa:
       - sms
+      - phone
```

Comment:
Is this from personal experience or is there some documentation that they support 2FA through phone calls which I can't find?

Reply:
This is from personal experience. After clicking the "I need help getting a verification code" link, the following screen appears for me:
![evernote-tfa](https://user-images.githubusercontent.com/520841/71157824-09c0b980-2243-11ea-8fa6-d77c49642f6d.png)
I don't know whether voice call is supported everywhere, or why they didn't include it in their docs. But it did work.